strawberryfields/apps/graph/resize.py

```diff
@@ -308,6 +308,24 @@ def clique_swap(clique: list, graph: nx.Graph, node_select: str = "uniform") ->
     supported. Degree-based node selection involves picking the node with the greatest degree,
     with ties settled by uniform random choice.
 
+    Example usage:
+
```

Comment:
I also recommend adding the `sphinx_autodoc_typehints` extension to Sphinx, any objections?

This is how things render **without** the extension:
![Annotation 2019-08-30 153515](https://user-images.githubusercontent.com/49409390/64047200-24f4a580-cb3c-11e9-8021-e794ef35526d.png)

and this is how things render **with**:
![Annotation 2019-08-30 153552](https://user-images.githubusercontent.com/49409390/64047227-3342c180-cb3c-11e9-8f1f-4207d0281b17.png)

There is also a https://pypi.org/project/sphinx-autodoc-napoleon-typehints/ but I couldn't get it working.

Reply:
That's weird, why does the argument to `graph` display a `MagicMock` object in the first instance?

Reply:
I'm in favour of adding this extension. The one thing to double check is if it affects the docstrings of existing SF documentation.

Reply:
I have added in!

So to check how it looks like with the rest of the SF documentation (TLDR: I didn't see any difference!), here is a screenshot **WITH** `sphinx_autodoc_typehints`:

![Annotation 2019-09-03 123027](https://user-images.githubusercontent.com/49409390/64191608-d6e6e700-ce46-11e9-817c-33aa09ac182e.png)



and **WITHOUT**:


![Annotation 2019-09-03 123046](https://user-images.githubusercontent.com/49409390/64191616-dcdcc800-ce46-11e9-9972-a761ecc84a6a.png)

Reply:
Also I don't see any warnings during `make docs`.